e2e/browser/testApp/components/appContent/index.jsx

```diff
@@ -87,7 +87,7 @@ export default function Home() {
       return;
     }
     (async () => {
-      const accessGrant = await approveAccessRequest(
+      const accessGrantRequest = await approveAccessRequest(
```

Comment:
Approving the request issues the actual access grant, so I'd keep the initial naming.

Reply:
This change was because `accessGrant` is declared in the upper scope (no-shadow)